packages/next/src/build/index.ts

```diff
@@ -357,12 +357,10 @@ async function writeEdgePartialPrerenderManifest(
       dynamicRoutes: {},
       notFoundRoutes: [],
       version: manifest.version,
-      preview: {
-        previewModeId: 'process.env.__NEXT_PREVIEW_MODE_ID',
-        previewModeSigningKey: 'process.env.__NEXT_PREVIEW_MODE_SIGNING_KEY',
-        previewModeEncryptionKey:
-          'process.env.__NEXT_PREVIEW_MODE_ENCRYPTION_KEY',
-      },
+      // Preview props are inlined in the code with dynamic env vars,
```

Comment:
These values were the main reason this manifest was added iirc so should we just fully remove it? 

Could be in a separate PR just noting for cleanup purposes.

Reply:
Sounds good, I'll get a follow up PR to remove it from edge runtime